sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -243,18 +242,17 @@ - (void)dealloc {
   [self.player destroy];
 }
 
-@end
-
-@implementation OOSkinViewController(Internal)
-
-- (void)toggleFullscreen {
+- (void)setFullscreen:(BOOL)fullscreen {
+  if (fullscreen == _fullscreen) return;
```

Comment:
Can we check against the video player isFullscreen instead? This way we can have one source of truth.